public/index.html

```diff
@@ -7,6 +7,50 @@
   </head>
 
   <body>
+    <div id="main"> 
```

Comment:
- [ ] Please change this `div` to a `main` tag, using the `main` tag is highly recommended as it follows semantic HTML practices and enhances the readability of your code. It basically shows that this is the main area of your web page content separating it from the `header` and  `footer`.